scripts/calc_var.py

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env python
+#! /usr/bin/env python3
```

Comment:
Why are you changing these to python3? The setup.py should do its python magic to change this

Reply:
I was seeing some inconsistency between files in how the shebang was formatted so I wanted to be consistent. 

I was also seeing that this line was occasionally changed to be `!#/usr/bin/python` or some other not-useful thing when being installed in the docker container. This would then cause "bad interpreter" errors to happen when running the pipeline.

I've read that `python3` should always map to python version 3, where as python can map to either 2 or 3 depending on the install. Is specifying `python3` going to cause issues?

Reply:
As long as you install the python scripts with setup.py, the shebang will be changed the directory of the python executable. So python vs python3 will only matter if you're running the scripts like `./calc_var.py` so if you think python3 is safer it's fine by me

Reply:
Ok good to know. Let's leave it as `python3` since these scripts are still used by people who can't "install" the code (eg, some users on Pawsey).